runtime/pom.xml

```diff
@@ -51,10 +51,20 @@
             <artifactId>api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
Runtime already includes slf4j impl, so the api is implicite there.

Reply:
removed.